var/spack/repos/builtin/packages/curl/package.py

```diff
@@ -239,3 +266,68 @@ def with_or_without_secure_transport(self, activated):
                 return "--with-darwinssl"
             else:
                 return "--without-darwinssl"
+
+    def configure(self, spec, prefix):
+        options = getattr(self, "configure_flag_args", [])
+        options += ["--prefix={0}".format(prefix)]
+        options += self.configure_args()
+
+        with working_dir(self.build_directory, create=True):
+            configure(*options)
```

Comment:
This looks similar to the default implementation (and likewise for `build`) - am I missing something you added or does a failure occur when you don't add these definitions?